roles/calibre-web/tasks/main.yml

```diff
@@ -0,0 +1,111 @@
+- name: Create calibre-web folders to store data and configuration files.
+  file:
+    path: "{{ item }}"
+    owner: "{{ calibreweb_user }}"
+    group: "{{ apache_user }}"
+    mode: 0755
+    state: directory
+  with_items:
+    - "{{ calibreweb_home }}"
+    - "{{ calibreweb_path }}"
+
+## TODO: Calibre-web future release might get into pypi https://github.com/janeczku/calibre-web/issues/456
+- name: Download calibre-web github repository.
+  git:
+    repo: https://github.com/janeczku/calibre-web.git
+    dest: "{{ calibreweb_path }}"
+    update: yes
+    depth: 1
+    version: master
+  when: internet_available
+
+## TODO: Use Ansible native pip module for this. How do you use --target vendor ?
+- name: Download calibre-web dependencies into vendor subdirectory.
+  command: pip install --target vendor -r ./requirements.txt
+  args:
+   chdir: "{{ calibreweb_path }}"
+
+- name: Create calibre-web systemd service unit and httpd2 configuration.
+  template:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    mode: "{{ item.mode }}"
+    owner: "{{ calibreweb_user }}"
+    group: "{{ apache_user }}"
+  with_items:
+    - { src: 'calibre-web.service.j2', dest: '/etc/systemd/system/calibre-web.service', mode: '0644' }
+    - { src: 'calibre-web.conf.j2', dest: '/etc/apache2/sites-available/calibre-web.conf', mode: '0644' }
+    - { src: 'cps.conf.j2', dest: "{{ calibreweb_home }}/cps.conf" , mode: '0644' }
+
+
+- name: Provision calibre-web with default administration account and database.
+  copy:
+    src: roles/calibre-web/files/metadata.db
+    dest: "{{ calibreweb_home }}/{{ calibreweb_database }}"
+    owner: "{{ calibreweb_user }}"
+    group: "{{ apache_user }}"
+    mode: 0644
+    backup: yes
+  when: calibreweb_provision
+
+- name: Ask systemd to reread unit files (daemon-reload).
```

Comment:
@jvonau reminds us that this daemon-reload needs to happen after unit file(s) are in place.  One way to fix it (among others, as was mentioned [last week](https://github.com/iiab/iiab/pull/999#discussion_r210510457)) would be to replace both stanzas (in lines 51-60) with this unified stanza:
```
- name: daemon-reload, enable & (re)start calibre-web service
  systemd:
    name: calibre-web
    daemon_reload: yes
    enabled: yes
    state: restarted
  when: calibreweb_enabled
```